classes/template/PKPTemplateResource.php

```diff
@@ -49,6 +52,20 @@ public function __construct($templateDir)
     public function fetch($name, &$source, &$mtime)
     {
         $filename = $this->_getFilename($name);
+
+        if ($this->isBladeViewPath($filename)) {
+            try {
+                $mtime = time();
+                $templateManager = TemplateManager::getManager(Application::get()->getRequest());
+                // $templateManager->shareTemplateVariables($templateManager->getTemplateVars());
+                $source = view($filename, $templateManager->getTemplateVars())->render();
+                return true;
+            } catch (Throwable $exceptin) {
+                error_log("Error rendering Blade view '{$filename}': " . $exceptin->getMessage());
+                throw $exceptin;
```

Comment:
*exception; also, is it necessary to include both an `error_log` and a `throw`? The `Throwable` should have enough detail in it already, I think.